packages/invoices/src/InboundShipment/DetailView/reducer/reducer.ts

```diff
@@ -1,3 +1,4 @@
+import { groupBy } from './../../../../../common/src/utils/arrays/utils';
```

Comment:
time to introduce `@common/utils` - if we weren't planning to replace with lodash 😃